arcgis-ios-sdk-samples/Edit data/Create and save KML file/CreateAndSaveKMLViewController.swift

```diff
@@ -61,8 +61,12 @@ class CreateAndSaveKMLViewController: UIViewController {
         let activityViewController = UIActivityViewController(activityItems: [kmzProvider], applicationActivities: nil)
         activityViewController.popoverPresentationController?.barButtonItem = sender
         present(activityViewController, animated: true)
-        activityViewController.completionWithItemsHandler = { (_, _, _, _) in
-            kmzProvider.deleteKMZ()
+        activityViewController.completionWithItemsHandler = { activityType, completed, returnedItems, activityError in
```

Comment:
The unused values should still be represented by an underscore.
```suggestion
        activityViewController.completionWithItemsHandler = { _, completed, _, activityError in
```